common-lib/src/main/scala/com/gu/mediaservice/lib/aws/S3.scala

```diff
@@ -208,9 +208,6 @@ class S3(config: CommonConfig) extends GridLogging {
     S3Metadata(meta)
   }
 
-  def getUserMetadata(bucket: Bucket, key: Key): Map[Bucket, Bucket] =
-    client.getObjectMetadata(bucket, key).getUserMetadata.asScala.toMap
-
```

Comment:
This seems unrelated?

Reply:
It was unused code.

Reply:
I prefer not to do housekeeping in files that are otherwise unchanged as it makes for unclean PRs.

Reply:
you can't win :(